docs/standard/threading/how-to-register-callbacks-for-cancellation-requests.md

```diff
@@ -1,28 +1,33 @@
 ---
-title: "How to: Register Callbacks for Cancellation Requests"
-ms.date: "03/30/2017"
+title: "How to: Register callbacks for cancellation requests"
+ms.date: 08/14/2020
+ms.topic: how-to
```

Comment:
I think this is unnecessary. It was defined globally in https://github.com/dotnet/docs/pull/19940

Reply:
That is only for articles that are in a "how-to" folder, this article is a how-to and needs to be explicitly specified as such.

Reply:
@IEvangelist From my understanding, it sets it for any file name that contains "how-to"